documentation/src/docs/deployment/run-local.md

```diff
@@ -7,25 +7,30 @@ You may obtain a copy of the License at
 
     http://www.apache.org/licenses/LICENSE-2.0
 -->
-# Running Pravega
+# Running Pravega in Local Machine
 
+As an alternative to running Pravega on a cluster of machines, you may run Pravega on a local machine. Running Pravega locally on a single host, allows you to get started with Pravega quickly. Running Pravega locally is especially suitable for development and testing purposes.
```

Comment:
remove comma between `host` and `allows`

Reply:
Done.